igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabProperties.java

```diff
@@ -16,48 +16,42 @@
 
 package com.netflix.spinnaker.igor.config;
 
+import javax.validation.constraints.NotNull;
 import org.hibernate.validator.constraints.NotEmpty;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
-import javax.validation.constraints.NotNull;
-
-/**
- * Helper class to map masters in properties file into a validated property map.
- */
+/** Helper class to map masters in properties file into a validated property map. */
 @ConditionalOnProperty("gitlab.base-url")
 @ConfigurationProperties(prefix = "gitlab")
 public class GitLabProperties {
-    @NotEmpty
-    private String baseUrl;
+  @NotEmpty private String baseUrl;
 
-    @NotEmpty
-    private String privateToken;
+  @NotEmpty private String privateToken;
```

Comment:
Are we putting all annotations on the same line? or just this one?

Reply:
It's the way Google's code format forces... so... yes?